modules/developer_manual/examples/scripts/backport.sh

```diff
@@ -21,11 +21,13 @@ fi
 
 commit=$1
 targetBranch=$2
+baseBranch=$(ggit rev-parse --abbrev-ref HEAD)
```

Comment:
what's `ggit`?

Reply:
:man_facepalming: thanks catching it, copy/paste error